samples/ContinuousIntegration/GitHubActions/0.62/build-nativemodule-ci.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will build all flavors of your React Native for Windows native
+#  module project every time changes are pushed to master. This checks that
+# every change builds on Windows.
+
+name: Native Module CI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  # This job builds both Debug and Release versions of the native module, for
+  # x86, x64, ARM and ARM64 architectures.
+  build:
+    name: Build
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        architecture: [x86, x64, ARM, ARM64]
+        configuration: [Debug, Release]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Yarn install
+        run: yarn install
+
+      - name: Build ${{ matrix.architecture }} Debug
+        if: ${{ matrix.configuration == 'Debug' }}
+        run: npx react-native run-windows --no-deploy --no-launch --no-packager --logging --arch ${{ matrix.architecture }}

```

Comment:
why no deploy (for x86/x64)? same below

Reply:
It's a native module, nothing to deploy.